pkg/errors/sdk_errors.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package errors
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+var (
+	ErrJSONKeyNotFound = errors.New("response body expected error message json key not found")
+	ErrUnknown         = errors.New("unknown error")
+)
+
+const err = "error"
```

Comment:
Improve the name, something like `jsonErrKey` would be better.

Reply:
I changed this to errJSONErrKey (added err prefix, since we're using it as an error now)